test/plugin/test_kafka_plugin_util.rb

```diff
@@ -35,4 +35,17 @@ def test_read_ssl_file_when_non_empty_array
         assert_equal(["a","b"], read_ssl_file(["a","b"]))
     end
 
-end
\ No newline at end of file
+    def test_read_ssl_ca_certs_when_nil
+      assert_equal(nil, read_ssl_ca_certs())
+    end
+
+    def test_read_ssl_ca_certs_general_case
+      begin_cert = "-----BEGIN CERTIFICATE-----\n"
+      end_cert = "\n-----END CERTIFICATE-----\n"
+      cert_one = begin_cert + "one" + end_cert
```

Comment:
I think only one line isn't suitable for a test.
Should use a more practical certificate string.

Reply:
A string actually generated by OpenSSL is desired.

Reply:
how does anything other then the start and end certificate add value in a unit test?  The beginning and end lines are the only thing of significance since everything in between is irrelevent

Reply:
At least intermediate data should be multi lines.
If we replace the implementation with `String#scan` and its regexp is inappropriate, it might not be able to match with multi lines even if it passes the test with a single line.